components/broadcast_talent_table/commons/broadcast_talent_table.lua

```diff
@@ -37,7 +37,8 @@ local DEFAULT_ABOUT_LINK = 'Template:Weight/doc'
 local ACHIEVEMENTS_SORT_ORDER = 'weight desc, date desc'
 local RESULTS_SORT_ORDER = 'date desc'
 
---- @class BroadcastTalentTable
+---@class BroadcastTalentTable
+---@operator call:BroadcastTalentTable
```

Comment:
```suggestion
---@operator call():BroadcastTalentTable
```
While not required, I think it's good to be specific here, as all other functions require it